src/components/Species/index.tsx

```diff
@@ -819,6 +820,13 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
             <Grid item xs={12}>
               {results && (
                 <Table
+                  setColumns={(columnsToSet: TableColumnType[]) => {
```

Comment:
species table overrides the implementation to set columns upon reshuffling (it filters out error column table if it is not being shown)